src/gbinder_servicemanager_aidl3.c

```diff
@@ -84,28 +76,6 @@ gbinder_servicemanager_aidl3_get_service(
     return obj;
 }
 
-static
-GBinderLocalRequest*
-gbinder_servicemanager_aidl3_add_service_req(
```

Comment:
What's the point in moving this function around?

Reply:
Just in compliance with the following order of the operations that are defined in other AIDL service managers
- get
- list
- add

It's just my personal preference. I can revert it back if you don't feel good about it.

Reply:
That's fine, then. Besides, you have written that code in the first place))